new-toast-site/src/pages/styles.js

```diff
@@ -1,6 +1,6 @@
 /** @jsx jsx */
 import { jsx } from "@emotion/core";
-import StyleGuideComponent from "../StyleGuideComponent.js";
+import StyleGuideComponent from "../temp/StyleGuideComponent.js";
```

Comment:
seemed like the import wasn't pointed at the right file here, is this change the intended file?

Reply:
hey chris, yeah for now my styleGuide component is living that temp directory. Once this Pr is merged, I'm going to clean my directories, b/c some of the folder names are not very helpful.